File: Functions/notes.c
// Gabriel Lopez, Functions Notes on C

#include <stdio.h>
int num;
int add(int numOne, int numTwo){
    return numOne+numTwo;
}
void due(char assignment[50], char day[20]){
printf("The %s assignment is due %s\n", assignment, day);
}

int main(void){
    printf("Please tell me a number:\n");
    scanf("%d", num);
    add(4,10);
    add(8,1);
    add(72,5);
    printf("%d", add(72,5));
    return 0;
    due("Functions Notes", "Today");
    due("Hello World Update", "Tomorrow");
    due("Financial Calculator Update", "Friday");
}

File: Functions/notes.py
# Gabriel Lopez, Functions Notes on Python

#functions hold actions to be reused 

#def add():
    #numOne= int(input("Please tell me a number:\n"))
    #numTwo= int(input("Please tell me a another number:\n"))
    #print(numOne+numTwo)

#number= int(input("Please tell me a number:\n")) #perameters set the name of the variable (just for the function)
#def add(numOne,numTwo):
    #return numOne + numTwo 

#print(add(number,21)) #arguments set the vaule of the variable just for that instance of the function
#print(add(8,12))
#add(6,4)

def values(type):
    return input(f"Please give me a {type}:\n")

name =values("name")
place = values("place")
verb = values("verb (past tense)")

print(f"{name} was really fast getting to {place} because they {verb} the whole way there.")